server/service/hosts.go

```diff
@@ -950,13 +976,27 @@ func hostsReportEndpoint(ctx context.Context, request interface{}, svc fleet.Ser
 		return hostsReportResponse{Err: err}, nil
 	}
 
-	// Those are not supported when listing hosts in a label, so that's just to
-	// make the output consistent whether a label is used or not.
-	req.Opts.DisableFailingPolicies = true
+	req.Opts.DisableFailingPolicies = false
```

Comment:
I don't follow the change from `true` to `false` for `DisableFailingPolicies`.

Reply:
We don't want to disable them, we need to failing counts (for the `issues` column).